compiler/rustc_parse/src/parser/diagnostics.rs

```diff
@@ -573,6 +574,13 @@ impl<'a> Parser<'a> {
             return Err(self.sess.create_err(UseEqInstead { span: self.token.span }));
         }
 
+        if self.token.ident().map(|(ident, _)| ident.as_str() == "move").unwrap_or_default()
```

Comment:
you should compare the `Symbol` to [`kw::Move`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_span/symbol/kw/constant.Move.html) instead of doing a string comparison

same for the `prev_token` below, using `token_descr` is not great

Reply:
thanks!